source/plugins/cloudstack-kubernetes-service.rst

```diff
@@ -194,6 +200,13 @@ New Kubernetes clusters can be created using the API or via the UI. User will be
 
 |cks-create-cluster-form.png|
 
+Since 4.21.0, users will be provided with an additional section displayed when toggling the option: 'Show Advanced Settings'. On this section, users can select templates and service offerings for:
```

Comment:
```suggestion
Since 4.21.0, users will be provided with an additional section displayed when toggling
the option: 'Show Advanced Settings'. On this section, users can select templates and
service offerings for:
```